### PR TITLE
Anas/EDLY-6266 Add course limit check and redirect to billing

### PR DESCRIPTION
**Pull request by Anas-hameed**
## Description
Allow only 5 courses creation on the studio in free trial and redirect the user to payment page incase user exceed the limit with appropriate Error messaging.

## Jira 
https://edlyio.atlassian.net/browse/EDLY-6266?atlOrigin=eyJpIjoiNWQ0OTJhNWIzNzk1NDQzYWEzYjgzZjA5YWU1NmFjMGYiLCJwIjoiaiJ9

### CODE REVIEW

**Review by taimoor-ahmed-1 — APPROVED** [review]

File: cms/djangoapps/contentstore/views/course.py [hunk 1]
```diff
@@ -627,6 +627,10 @@ def format_library_for_view(library):
 
     site_config = configuration_helpers.get_current_site_configuration()
     tracking_api_url = f"{site_config.get_value('PANEL_NOTIFICATIONS_BASE_URL')}/api/v1/tracking_events/"
+    frontent_redirect_url = ''
+    frontend_url = [url for url in settings.CORS_ORIGIN_WHITELIST if 'apps' in url]
```
Comment (muhammadali124) [inline cms/djangoapps/contentstore/views/course.py:631]: @taimoor-ahmed-1 Is there better way to get panel frontend url or should we go with this for now? @Anas-hameed and I went over the code and there currently panel frontend url is not available in studio

**Review by muhammadali124 — CHANGES_REQUESTED** [review]

File: cms/djangoapps/contentstore/views/course.py (hunk 1 above)
Comment (taimoor-ahmed-1) [reply cms/djangoapps/contentstore/views/course.py:631]: seems fine for now

Comment (muhammadali124) [reply cms/djangoapps/contentstore/views/course.py:631]: Alright 👍

**Review by muhammadali124 — APPROVED** [review]
LGTM

**Pull request merged**